bitwarden_license/bit-web/src/app/tools/access-intelligence/all-applications.component.ts

```
@@ -72,7 +74,8 @@ export class AllApplicationsComponent implements OnInit {
       FeatureFlag.CriticalApps,
     );
 
-    const organizationId = this.activatedRoute.snapshot.paramMap.get("organizationId");
+    const organizationId =
+      (this.activatedRoute.snapshot.paramMap.get("organizationId") as string) ?? "";
```

🤔 Is there ever a case where the page shouldn't be supplied an `organizationId`? If so, then you should conditionally run the logic that depends on `organizationId`. If not, then you should throw an error.

---

In fact, it was this line that caused tests to fail. Turns out that `paramMap.get('..')` returns `string | undefined` - and that caused the tests to fail. This piece of code fixed it. This is the link to the failing tests
https://github.com/bitwarden/clients/actions/runs/12794336886/job/35669250167

---

I have fixed this to remove the type.